api/app.js

```diff
@@ -14,6 +15,15 @@ module.exports = {
         app.post('/sendCode', authController.sendCode);
```

Comment:
Create routes directory for it. [Example](https://github.com/AlSoEdit/telegram27/tree/master/routes).